v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+| id          | [`INT8`](int.html)            | The ID of the report run that generated the data available in the `system.replication_*` tables. |
+| generated   | [`TIMESTAMP`](timestamp.html) | When the report was generated, in UTC.                                                           |
+
+### crdb_internal.zones
+
+The `crdb_internal.zones` table is useful for:
+
+- Matching up data returned from the various conformance reports with the human-readable names of the databases and tables, indexes, and partitions where that data lives.
+- Viewing your cluster's zone configurations in various formats: YAML, SQL, etc.
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM crdb_internal.zones;
+~~~
+
+| column_name         | data_type               | description                                                                                                          |
+|---------------------+-------------------------+----------------------------------------------------------------------------------------------------------------------|
+| zone_id             | [`INT8`](int.html)      | The ID of the [replication zone](configure-zone.html).                                                               |
+| subzone_id          | [`INT8`](int.html)      | The ID of the subzone (i.e., [partition](partition-by.html)).                                                        |
+| target              | [`STRING`](string.html) | The "object" that the constraint is being applied to, e.g., `PARTITION us_west OF INDEX movr.public.users@primary`.  |
+| range_name          | [`STRING`](string.html) | The human-readable printed representation of the range. Unimplemented.                                               |
+| database_name       | [`STRING`](string.html) | The [database](show-databases.html) where the `target`'s data is located.                                            |
+| table_name          | [`STRING`](string.html) | The [table](show-tables.html) where the `target`'s data is located.                                                  |
+| index_name          | [`STRING`](string.html) | The [index](show-index.html) where the `target`'s data is located.                                                   |
+| partition_name      | [`STRING`](string.html) | The [partition](show-partitions.html) where the `target`'s data is located.                                          |
+| raw_config_yaml     | [`STRING`](string.html) | The YAML you used to [configure this replication zone](configure-replication-zones.html).                            |
+| raw_config_sql      | [`STRING`](string.html) | The SQL you used to [configure this replication zone](configure-replication-zones.html).                             |
+| raw_config_protobuf | [`BYTES`](bytes.html)   | The protobuf representation of the YAML used to [configure this replication zone](configure-replication-zones.html). |
```

Comment:
I think some of these columns are hidden, aren't they? (i.e. they don't show up in select *). I wouldn't document those.

Reply:
I'll remove the `raw_*` columns but FWIW they do show up in the default output of

`SELECT * FROM crdb_internal.zones;`

It returns the columns

- zone_id
- subzone_id
- target
- range_name
- database_name
- table_name
- index_name
- partition_name
- raw_config_yaml
- raw_config_sql
- raw_config_protobuf
- full_config_yaml
- full_config_sql 

(This is in version `  CockroachDB CCL v19.2.0-rc.1 (x86_64-apple-darwin14, built 2019/10/14 18:28:24, go1.12.5)  `)